ecl/compute/v2/_proxy.py

```diff
@@ -22,6 +22,7 @@
 from ecl.compute.v2 import server_volume as _server_volume
 from ecl.compute.v2 import quota as _quota
 from ecl.compute.v2 import volume as _volume
+from ecl.image.v2 import image as _img
```

Comment:
外部のパッケージからimportしているのでasを使わなくて良い気がします。
```suggestion
from ecl.image.v2 import image
```

Reply:
ありがとうございます。修正を確認しました。